utilities_common/cli.py

```diff
@@ -5,7 +5,7 @@
 
 import click
 import json
-import netaddr
+#import netaddr
```

Comment:
how much delay is this one? btw, we moved to python3, is this still an issue?

Reply:
takes around 200ms